doc/cug.tex

```diff
@@ -5821,25 +5870,46 @@ \subsection{Suite And Task Event Handling}
 See also~\ref{SuiteEventHandling} and~\ref{TaskEventHandling}
 in the {\em Suite.rc Reference}.
 
-Cylc can call nominated event handlers when certain suite or task events
-occur. This is intended to facilitate centralized alerting and automated
-handling of critical events. Event handlers can send an email or an SMS,
-call a pager, and so on; or intervene in the operation of their own
-suite using cylc commands.  \lstinline=cylc [hook] email-suite= and
-\lstinline=cylc [hook] email-task= are example event handlers packaged
-with cylc.
+Cylc can call nominated event handlers when certain suite or task events occur.
+This is intended to facilitate centralized alerting and automated handling of
+critical events. Event handlers can be used to send a message, call a pager,
+and so on; or intervene in the operation of their own suite using cylc
+commands.
 
-Event handlers can be located in the suite bin directory, otherwise
-it is up to you to ensure their location is in \lstinline=$PATH=
-(in the shell in which cylc runs, on the suite host).
-
-Task event handlers are passed the following arguments by the suite daemon:
+To send an email, you can use the built-in setting
+\lstinline=[[[events]]]mail events= to specify a list of events for which
+notifications should be sent. E.g. to send an email on (submission) failed and
+retry:
 
+\lstset{language=suiterc}
 \begin{lstlisting}
-<task-event-handler> EVENT SUITE TASKID MESSAGE
-\end{lstlisting}
-where EVENT is one of the following:
-
+[runtime]
+    [[foo]]
+        retry delays = PT0S, PT30S
+        script = test "${CYLC_TASK_TRY_NUMBER}" -eq 3
+        [[[events]]]
+            mail events = submission failed, submission retry, failed, retry
+\end{lstlisting}
+
+By default, the emails will be sent to the current user with \lstinline=from:=
+set to \lstinline=notifications@$(hostname)= using an SMTP server at
+\lstinline=localhost:25=. These can be configured using the settings
+\lstinline=[[[events]]]mail to= (list of email addresses),
+\lstinline=[[[events]]]mail from= and \lstinline=[[[events]]]mail smtp=.
+
+Event handler commands can be located in the suite \lstinline=bin/= directory,
+otherwise it is up to you to ensure their location is in \lstinline=$PATH= (in
+the shell in which cylc runs, on the suite host). The commands should require
+very little resource to run and should return instantaneously. (Each event
```

Comment:
(I still don't like "instantaneously" here - it might suggest to users that event handlers should return immediately after spawning some job in the background ... "quickly" should do, no?)